modules/imapauth/docs/imapauth.txt

```diff
@@ -0,0 +1,48 @@
+ImapAuth module
```

Comment:
This could also be the `README.md` in the top of your repository, so that users can see how to install and use your module at first sight. Pretty much like [others](https://github.com/NIIF/simplesamlphp-module-attributeaggregator) are doing.